dolphinscheduler-task-plugin/dolphinscheduler-task-pytorch/src/test/java/org/apache/dolphinscheduler/plugin/task/pytorch/PytorchTaskTest.java

```diff
@@ -72,6 +72,12 @@ public void testPythonEnvManager() {
 
     }
 
+    @Test
+    public void testGitProjectUrlInjection(){
+        Assertions.assertFalse(GitProjectManager.isGitPath("git@& cat /etc/passwd >/poc.txt #"));
+        Assertions.assertFalse(GitProjectManager.isGitPath("git@| cat /etc/passwd >/poc.txt #"));
```

Comment:
Please add a positive case.

Reply:
> Please add a positive case which will return true.

Thank you for your reminder. I found that there are already positive test cases during testing.

 [exists test cases](https://github.com/apache/dolphinscheduler/blob/d218b021e7e045e7982a3c91ea7b1b1491d21d38/dolphinscheduler-task-plugin/dolphinscheduler-task-pytorch/src/test/java/org/apache/dolphinscheduler/plugin/task/pytorch/PytorchTaskTest.java#L76)

Reply:
Great

Reply:
> Great, please use `mvn spotless:apply` to format the code style.

Thank you for your advice,I have already resolved it.